src/front/components/vehicles/content.vue

```diff
@@ -27,10 +26,11 @@ function changeTab(index: number) {
         </li>
       </ul>
     </div>
-      <div v-for="(content, index) in tabContents" :key="index" v-show="activeTab === index">
-        <!-- Contenu dynamique de l'onglet -->
-        {{ content }}
-        <ListCard></ListCard>
+      <div v-if=useVehiclesListStore().isLoaded v-for="(_, index) in tabContents" :key="index" v-show="activeTab === index" class="h-125 overflow-y-scroll">
```

Comment:
Attention, il faut éviter d'utiliser v-if et v-show sur le même élément 
https://vuejs.org/guide/essentials/conditional.html#v-if-with-v-for